packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/QuickActions.tsx

```diff
@@ -24,36 +21,42 @@ const DescreetContainer = styled(NavigationItemBase)`
     }
 `;
 
-const DesktopContainer = styled(WebContainer)`
+const ActionsContainer = styled.div<{ numberOfColumns: number }>`
     position: relative;
     display: grid;
-    grid-template-columns: 1fr 1fr;
+    grid-template-columns: ${({ numberOfColumns }) => '1fr '.repeat(numberOfColumns)};
     gap: ${spacingsPx.md};
     padding: ${spacingsPx.xs};
     border-top: 1px solid ${({ theme }) => theme.borderOnElevation0};
+    align-items: stretch;
+`;
 
-    ::after {
+const Column = styled.div`
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    &:not(:last-child)::after {
```

Comment:
I believe vertical separators are there no more? Maybe `Column` will not be even needed and styling can be simplified even further! 🤩

<img width="366" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/9f7ad589-4579-41e1-b3c8-c4dcc479c5cb">

Reply:
Damn 😂